pcfuncs/animation/stamps/brand.py

```diff
@@ -0,0 +1,42 @@
+from PIL import Image, ImageDraw, ImageFont
+from PIL.Image import Image as PILImage
+
+from .stamp import TRANSPARENT, FrameStamp
+
+BOTTOM_OFFSET = 16
+PADDING = 2
+
+
+text = "planetarycomputer.microsoft.com"
+font = ImageFont.truetype("/usr/share/fonts/dejavu/DejaVuSans.ttf", 12)
```

Comment:
This is the only font family installed on the Azure Functions linux runtime container locally, but I need to verify it exists in the service container as well after it's deployed.